src/Snippets/Docs/Chaos.Outcome.cs

```diff
@@ -82,7 +85,7 @@ public static void OutcomeGenerator()
                 OutcomeGenerator = new OutcomeGenerator<HttpResponseMessage>()
                     .AddResult(() => new HttpResponseMessage(HttpStatusCode.InternalServerError)) // Result generator
                     .AddResult(() => new HttpResponseMessage(HttpStatusCode.TooManyRequests), weight: 50) // Result generator with weight
-                    .AddResult(context => CreateResultFromContext(context)) // Access the ResilienceContext to create result
+                    .AddResult(context => new HttpResponseMessage(CreateResultFromContext(context))) // Access the ResilienceContext to create result
```

Comment:
The usage samples advocate to use one of the anti-patterns

```cs
...
.AddResult(context => new HttpResponseMessage(CreateResultFromContext(context))) // Access the ResilienceContext to create result
.AddException<HttpRequestException>(), // You can also register exceptions
```

Please update these `OutcomeGenerator`s by removing the`AddException` clauses.

Reply:
When I originally opened the PR I had removed those guys since I had deprecated it, however, I put them back so that we support that scenario (injecting results and exceptions in the same strategy) but under their judgment after knowing the caveats/side effects explained in the anti-pattern sections and in the footer. I can remove that entirely from the docs but it would be hidden. Honestly, while I understand it is handy it feels like we're kinda discouraging people to use the right strategy to inject faults. 

We could also add a comment in the samples as I did in some snippets, like this: // ⚠️ Avoid this ⚠️ 
and maybe add a `remarks` on the API pointing to the docs? thoughts?